conf.py

```diff
@@ -103,3 +104,96 @@
                                   "README.html")): # don't create dangling symlinks;
                                                    # it annoyes Github Pages.
                 os.symlink("README.html", z)
+
+
+
+
+# -- Custom scripts ----------------------------------------------------------
```

Comment:
This line should be above

Reply:
thanks, fixed in https://github.com/neuropoly/neuro.polymtl.ca/pull/19/commits/176ebe57d49e93a0e5e362eaf85b2f26e02475a4